### DISCUSSION TITLE
Suddenly unable to detect tables the same

### DISCUSSION THREAD

**Discussion by dclapham** [jsvine/pdfplumber#355] [Ask for help with specific PDFs]
I was working with pdfplumber yesterday to extract data from a pdf.  Originally the table extraction picked up the following:

![image](https://user-images.githubusercontent.com/47010699/108511243-8f148380-72b7-11eb-9958-433b6d5ad514.png)

I managed to get the content extracted by placing explicit lines down the edge of the tables.  It was working again this morning. 
 However, I reset the kernal in Jupyter and now it is no longer detecting the lines for the rectangle around the purple cells:

![image](https://user-images.githubusercontent.com/47010699/108511449-d6027900-72b7-11eb-8df9-ad19c8f3045b.png)

I cant figure out what I did to get it working originally (and obvioulsy not saved the code for!)

**Comment by dclapham** [comment]
So after a bit of tinkering, I've found that what was picked up as rects yesterday is showing as curves today.  I've now added curves as explicit_horizontal_lines and its working.  Not sure why the change in how pdfplumber is seeing the pdf though.

**Comment by jsvine** [comment]
Thanks for the update. My best guess is that, between those screenshots, the version of `pdfminer.six` changed. (That's what pdfplumber depends on to detect rects/curves/etc., and the particular logic of that occasionally changes between their library versions.) Do you recall upgrading or downgrading it?

**Reply by dclapham** [reply]
Its possible, I did update the packages in Anaconda whilst trying various pdf table extractors but not sure if it was before or after using pdfplumber.  Thanks for the suggestion though.  I'll keep an eye on that in future.
